src/integration-tests/test_connection_loss.py

```diff
@@ -0,0 +1,202 @@
+# Copyright 2024 Bloomberg Finance L.P.
+# SPDX-License-Identifier: Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+This suite of test cases exercises connection losses.
+"""
+
+import json
+import re
+from typing import Dict
+
+from blazingmq.dev.it.fixtures import (  # pylint: disable=unused-import
+    Cluster,
+    order,
+    cluster,
+    multi_node,
+    tweak,
+    start_cluster,
+)
+from blazingmq.dev.it.process.admin import AdminClient
+from blazingmq.dev.it.process.client import Client
+from blazingmq.dev.it.process.proc import Process
+
+pytestmark = order(1)
```

Comment:
```suggestion
pytestmark = order(2)
```

The `order` constitutes a logical dependency of tests, where only a higher level test can depend on the success of a lower level test. Now, `test_breathing.py` has order(1) and `test_restart.py` has order(2). I believe this test should be at least order(2).